grammar.js

```diff
@@ -285,6 +304,7 @@ module.exports = grammar({
         $._insert_statement,
         $._update_statement,
       ),
+      optional($.window_clause),
```

Comment:
I think this is the right place, since iirc window functions can be used with updates at least